### DISCUSSION TITLE
Can I host some tree-sitter here?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [tree-sitter-grammars/.github#10] [Transfer]
I created [some tree-sitter grammars](https://github.com/Freed-Wu?tab=repositories&q=tree-sitter) recently in order to create some language servers and I ask the upstream (that is XXX for XXXrc) and they seem to be not very interested to them. Can I host them here?

Some difference between them and [template](https://github.com/tree-sitter-grammars/template) is I add a python binding in `bindings/python` which use cmake and scikit-build-core to build python wheel. I think it should not be a problem.

**Comment by ObserverOfTime** [comment]
We will not be accepting any transfer requests at this time while we work on standardizing our current parsers.
Once this process is finished, you may review the [contributing guidelines] and reopen this discussion.

[contributing guidelines]: https://github.com/tree-sitter-grammars/.github/blob/main/CONTRIBUTING.md

**Reply by cathaysia** [reply]
what's mean of `while we work on standardizing our current parsers`? tree-sitter itself not been stabilized yet?

**Reply by clason** [reply]
No, as you can see from its repo. One of the things in the pipeline is support for packaging and publishing parsers through the tree-sitter CLI. The goal is to add workflows for that and polish the ~80 parsers we maintain first; once these are shipshape, we'll consider adding newer ones (and giving them the same treatment).

**Reply by Freed-Wu** [reply]
> we work on standardizing our current parsers

Any progress about it? There are many [transfer requests](https://github.com/tree-sitter-grammars/.github/discussions?discussions_q=label%3Atransfer) wait it.

**Reply by clason** [reply]
No, that process is stalled because there are too many parsers already. Going forward, we need to be very selective and likely not accept third-party submissions at all.

**Reply by Freed-Wu** [reply]
> not accept third-party submissions at all.

How about a new organization named tree-sitter-community to collect those unofficial projects related to tree-sitter? Any user who use those grammars from tree-sitter-community must know it is **unofficial** and security and quality will not be ensured. And those grammars in this organization is official, the security and quality can be ensured.

**Reply by clason** [reply]
Go ahead. I have no interest in maintaining that.
